packages/components/src/MultiSelect/subcomponents/MultiSelectToggle/_docs/MultiSelectToggle.stickersheet.stories.tsx

```diff
@@ -91,6 +91,46 @@ const StickerSheetTemplate: StickerSheetStory = {
           </StickerSheet.Body>
         </StickerSheet>
 
+        <StickerSheet
+          isReversed={isReversed}
+          heading="Validation states"
+          className="w-full"
+        >
+          <StickerSheet.Header
+            headings={["default", "Hover", "Focus"]}
+            hasVerticalHeadings
+            verticalHeadingsWidth="10rem"
+          />
+          <StickerSheet.Body>
+            <StickerSheet.Row rowTitle="Error">
+              <MultiSelectToggle status="error" {...defaultProps} />
+              <MultiSelectToggle
+                status="error"
+                {...defaultProps}
+                data-sb-pseudo-styles="hover"
+              />
+              <MultiSelectToggle
+                status="error"
+                {...defaultProps}
+                data-sb-pseudo-styles="focus"
+              />
+            </StickerSheet.Row>
+            <StickerSheet.Row rowTitle="Caution">
+              <MultiSelectToggle status="caution" {...defaultProps} />
+              <MultiSelectToggle
+                status="caution"
+                {...defaultProps}
+                data-sb-pseudo-styles="hover--button"
+              />
+              <MultiSelectToggle
+                status="caution"
+                {...defaultProps}
+                data-sb-pseudo-styles="focus--button"
```

Comment:
You could argue that the hover and focus change is tested in the previous story since our border don't change for validation states. Though I do wonder if there's value in it to ensure that there is a11y contrast in the story.

Reply:
Maybe I'll remove them and we can add the hover and focus states back into the Toggle if ever add any additional styles (like a darkened border)

Reply:
Yeah I think if they don't have anything specific, it should be safe to not have them :)